debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/xstream/LcrEventHandler.java

```diff
@@ -234,7 +235,8 @@ private void dispatchSchemaChangeEvent(DDLLCR ddlLcr) throws InterruptedExceptio
                         ddlLcr.getDDLText(),
                         schema,
                         ddlLcr.getSourceTime().timestampValue().toInstant(),
-                        streamingMetrics));
+                        streamingMetrics,
+                        null));
```

Comment:
Do we know whether Xstream provides the `TRUNCATE` DDL in the event stream?  If so, shouldn't we be able to also support this feature when Xstream is enabled instead of LogMiner?

Reply:
will add xstream support in next version of this PR